src/Avalonia.Visuals/Visual.cs

```diff
@@ -377,7 +377,7 @@ static void InvalidateAndSubscribe(AvaloniaPropertyChangedEventArgs e)
             }
         }
 
-        protected override void LogicalChildrenCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
+        protected override void LogicalChildrenCollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
```

Comment:
Not sure why this one only started showing up now, but fixed it in this PR anyway.